CHANGELOG.rst

```diff
@@ -39,6 +39,9 @@ Changelog
   X.509 extension.
 * Added support for parsing
   :class:`~cryptography.x509.SignedCertificateTimestamps` in OCSP responses.
+* X.509 certificate parsing now enforces that the ``version`` field contains a
```

Comment:
We should probably flag this as backwards incompatible right? Also, if you put this with the other incompatible entries we'll avoid a conflict with another PR landing 😆

Reply:
My thought was that it's such a minor change, there was no need to call it out in bold. WDYT?

Reply:
I think people use us in parsers that scrape every cert they can find so this will potentially break them and we should call it out even though it's hypothetically a very rare case.